runtime/src/main/codestarts/quarkus/quakus-hilla-maven-codestart/codestart.yml

```diff
@@ -0,0 +1,20 @@
+name: quarkus-hilla-maven-codestart
+ref: quarkus-hilla-maven-starter
+type: code
+tags:
+  - extension-codestart
+  - maven-only
+
+metadata:
+  title: Quarkus-Hilla starter
+  description: Start to code with the Quarkus-Hilla extension.
```

Comment:
Some more meaningful description, refering hilla etc. ? Is this shown somewhere ?

Reply:
It is shown as a tooltip in code.quarkus.io, as in the screenshot below

![image](https://github.com/mcollovati/quarkus-hilla/assets/4648894/a4bb431e-4bc1-4014-87df-97e0b6abaf4e)

Please update it as with some additional info. But I suppose it should not be too verbose

Reply:
And from where comes the description, which is below ?